test/zipObj.js

```diff
@@ -11,9 +11,8 @@ describe('zipObj', function() {
     eq(R.zipObj(['a', 'b', 'c'], [1, 2, 3, 4, 5, 6, 7]), {a: 1, b: 2, c: 3});
   });
 
-  it('extra keys are undefined', function() {
```

Comment:
That means that we should leave it open a bit longer for discussion, and maybe dig a little into the history and talk to whoever implemented it.  I don't remember.  For all I know it was me.

I definitely prefer the altered behavior, but perhaps there was a good reason for the original one.  I'll try to look tonight.